code/src/main/kotlin/com/expediagroup/sdk/lodgingconnectivity/payment/PaymentClient.kt

```diff
@@ -38,6 +39,9 @@ import com.expediagroup.sdk.lodgingconnectivity.payment.operation.getPaymentInst
  * or timeouts.
  */
 class PaymentClient(config: ClientConfiguration) : GraphQLClient() {
+    init {
+        LodgingConnectivityLogMasking.enable()
```

Comment:
I think this should be initialized once in the underlying the `RequestExecutor` class, not in the domain clients.

Reply:
done.